Controller/DepartmentController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Controller;
+
+require_once __DIR__ . '../../Model/Department/Department.php';
+
+use Model\Department\DepartmentModel;
```

Comment:
sử dụng require_once __DIR__ . '../../Model/Department/Department.php';
và use Model\Department\DepartmentModel; có ý nghĩa gì

Reply:
sử dụng use Model\Department\DepartmentModel để  sử dụng class DepartmentModel trong namespace đã khai báo -> tạo đối tượng  DepartmentModel
require_once ý nghĩa tương tự
Nhưng khi sử dụng chỉ use Model\Department\DepartmentModel sẽ báo lỗi class DepartmentModel not found

Reply:
em thử đặt lại tên class và tên file giống nhau xem
use Model\Department\DepartmentModel;

Reply:
em đặt thử tên class với tên file giống nhau xem

Reply:
Em thử sửa nhưng không được ạ
https://stackoverflow.com/questions/12573808/php-namespaces-not-working